source/customizing_look_and_feel/03_adding_assets/assets.rst

```diff
@@ -1,13 +1,35 @@
 Adding assets
 =============
 
-When customizing your portal, you'll probably need static files (images, font files) to be available throughout the
-platform. This page will enable you to do just that: upload files to your platform so that you can link to them in the
-:ref:`stylesheet <theme__stylesheet>` or within your pages.
+Assets are static files that are uploaded in an Opendatasoft domain, in order to be usable throughout that same domain (e.g. in the domain :ref:`stylesheet <stylesheet>`, in :doc:`content pages </creating_content/index>`, etc.). Either images or fonts files can be added as assets in the domain.
```

Comment:
I think that the original 'static files' may be to technical and can be remove (keep only files)